### ISSUE TITLE
bootstrap script improvements

### ISSUE THREAD

**Issue by dorin100** [input-output-hk/jormungandr#469]
**Describe the bug**
1. automatically delete the storage file (when running bootstrap script - https://github.com/input-output-hk/jormungandr/blob/master/scripts/bootstrap)
2. add option/parameter to start the node with Faucet = UTXO Address

**To Reproduce**
1.  if you run the script 2 times and start the node, the second time you start the node, you will receive the below error:
```D:\iohk\otherProjects\jormungandr\scripts\boostrap>jormungandr --genesis-block block-0.bin --config config.yaml --secret pool-secret1.yaml
Jun 12 17:27:01.704 INFO storing blockchain in '"/d/iohk/otherProjects/jormungandr/scripts/boostrap/jormungandr-storage-test/blocks.sqlite"', task: init
Jun 12 17:27:01.729 INFO restoring state at tip 34c806dc532b84cb09a3d4ce9a4f3f2eb12773e0d9cd12391d23d251666bd434, task: bootstrap
Error while loading the blockchain state
Error in the blockchain storage: block not found
```

**Expected behavior**
1. you should not receive any error if you run the script multiple times, each time starting the node too. Basically, you do the same operations (run script, start node) so you should see the same result (node started successfully) 
2. besides the apiPort and consensus protocol, the Faucet Address Type would be a very useful parametrization for the end users.

**Additional context**

**Comment by vincenthz** [comment]
happy to receive PR for improvement like this

**Comment by onicrom** [comment]
I'll take a stab at this tonight after work

**Comment by NicolasDP** [comment]
fixed by #498
